src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Dsa.cs

```diff
@@ -167,7 +168,7 @@ internal static DSAParameters ExportDsaParameters(SafeDsaHandle key, bool includ
             int gLength,
             byte[] y,
             int yLength,
-            byte[] x,
+            byte[]? x,
```

Comment:
Why is x the only one that's nullable?  Is it fundamentally different in that regard from p, q, g, and y?

Reply:
Yeah, it's the private key :smile:

Reply:
Intuitive naming 😦